### PR TITLE
[4.0][NO CACHE] Convert Bootstrap tabs to CE

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue #22793 .

### Summary of Changes
No boottrap interactive components...

### Testing Instructions
Apply patch and check that the Global configuration page is working as intended

### CODE REVIEW

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 9e42f15297f0ff10bf8283349ccc5226020f36c3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23780">issues.joomla.org/tracker/joomla-cms/23780</a>.</sub>

**Review by C-Lodder — COMMENTED** [review]
Still have JHtml calls which need replacing with HTMLHelper

**Comment by dgrammatiko** [comment]
HAHAHA, I'm lazy and I copy pasted the lines from com content, didn't realised that were mixed...

**Comment by C-Lodder** [comment]
@dgrammatiko You have a PHPCS Drone error

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on b5fa3056da034ea3aec9a1bb250fe849f8ec1f3c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23780">issues.joomla.org/tracker/joomla-cms/23780</a>.</sub>

File: administrator/components/com_config/tmpl/component/default.php [hunk 1]
```diff
@@ -72,6 +53,9 @@
 								$groupClass = $field->type === 'Spacer' ? ' field-spacer' : '';
 							?>
 							<?php if ($field->showon) : ?>
+							<?php
+								// @todo restore controlling the tab with showon, currently not supported by joomla-tabs
```
Comment (HLeithner) [inline administrator/components/com_config/tmpl/component/default.php:57]: is there already a PR for this?

Comment (dgrammatiko) [reply administrator/components/com_config/tmpl/component/default.php:57]: No, the tabs script in its current state is not production ready neither feature complete. Eg I had no clue that Joomla was controlling tabs through showon...

There is a new version of the tabs (and all the other Custom Elements) that will be published in the new repo as soon as we clean up the custom elements in this repo

**Comment by dgrammatiko** [comment]
@wilsonge ??

**Comment by wilsonge** [comment]
What's the status on the new tabs code that replaces the existing missing functionality?

**Comment by dgrammatiko** [comment]
That functionality is useless, the new tabs support the disabled attribute per tab which is essentially the same thing so..

**Comment by wilsonge** [comment]
OK Well in that case you can remove the todo and commented out code if it has the same impact. Then this needs a test to ensure that is the case.

**Pull request closed without merging**